tools/omero/omero_import.xml

```diff
@@ -25,8 +25,8 @@
         omero import folder -T Dataset:name:$dataset_name
             -s $omero_host
             -p $omero_port
-            -u $__user__.extra_preferences.get('omero|username', $test_username)
```

Comment:
We need to bump the version suffix of the tool.

Reply:
Done